src/App/screens/Instructor/screens/GetPublished/index.js

```diff
@@ -1,100 +1,43 @@
-      description: 'Iterate with mentor until lesson is approved',
-    },
-    {
-      isComplete: includes(['published'], instructor.firstLessonState),
-      description: 'Publish lesson',
-    },
-  ]
-
-  const questions = [
-    {
-      question: 'How much money will I make?',
-      answer: 'Royalties can range anywhere from pocket change to surpassing your full-time job\'s income. It all depends on how many lessons and courses you create all your hard work snowballs into beautiful recurring residual income!',
-      moreInfoUrl: 'https://instructor.egghead.io/01-invited/getting-paid.html',
-    },
-    {
-      question: 'When do I get paid?',
-      answer: 'After you publish your first lesson, at the beginning of the following month we will set up your direct deposit.',
-    },
-    {
-      question: 'Can I see viewer stats for my lessons?',
-      answer: 'Yep! After you publish your first lesson, you will start getting nightly emails with viewer stats on your published lessons.',
-    },
-  ]
-
-  return (
-    <div>
-
-      <header>
-        <h1 className='f1'>Get Published</h1>
-      </header>
-
-      <div className='flex-ns'>
-
-        <main
-          className='mr4-ns'
-          style={{flex: 1}}
-        >
-          <h2 className='f2'>To Do</h2>
-          <DescriptionBlock>
-            Hey {instructor.first_name}! Work with your mentor to complete these items so you can get published!
-          </DescriptionBlock>
-          <div className='mt3'>
-            <Checklist items={todos} />
-          </div>
-        </main>
-
-        <aside style={{flex: 1}}>
-          <h2 className='f2'>FAQ</h2>
-          {questions.map((question, index) => (
-            <div key={index}>
-              <InfoBlock
-                title={question.question}
-                description={question.answer}
-                moreInfoUrl={question.moreInfoUrl}
-              />
-            </div>
-          ))}
-        </aside>
-
-      </div>
-
-    </div>
-  )
+import Split from './components/Split'
+import GetPublishedSteps from './components/GetPublishedSteps'
+import GetPublishedFaq from './components/GetPublishedFaq'
+
+class GetPublished extends React.Component {
+
+  fetchLessons(instructor) {
+    const {requestInstructorLessons} = this.props
+    requestInstructorLessons({
+      lessons_url: instructor.lessons_url,
+      page: 1,
+      size: 10
+    })
+  }
+
+  componentWillMount() {
+    const {instructor} = this.props
+    if(instructor) {
+      this.fetchLessons(instructor)
+    }
+  }
+
+  render() {
+    const {instructor, instructorLessons} = this.props
+    return (
+      <Split
+        title='Get Published'
+        main={
+          <GetPublishedSteps
+            instructor={instructor}
+            instructorLessons={instructorLessons}
+          />
+        }
+        aside={
+          <GetPublishedFaq />
+        }
+      />
```

Comment:
I would imagine many of our routes might be able to use this `Split` component, depending on how we wireframe them